src/Library/demos/Column View/main.js

```diff
@@ -0,0 +1,101 @@
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio";
+
+const column_view = workbench.builder.get_object("column_view");
+
+function createFirstCol() {
+  const factory = new Gtk.SignalListItemFactory();
+  const columnViewColumn = new Gtk.ColumnViewColumn({
+    factory,
+  });
+  factory.connect("setup", (factory, listItem) => {
+    const label = new Gtk.Label({
+      height_request: 50,
+      margin_start: 12,
+      margin_end: 12,
+    });
+    listItem.set_child(label);
+  });
+  factory.connect("bind", (factory, listItem) => {
+    const labelWidget = listItem.get_child();
+    const modelItem = listItem.get_item();
+    labelWidget.label = modelItem.get_name();
```

Comment:
Let's use `Gio.FileInfo.get_display_name()`, which may use a more human-readable name and then fallback to the standard `name` if necessary.